packages/discovery-provider/plugins/notifications/src/sns.ts

```diff
@@ -145,11 +145,7 @@ export const sendPushNotification = async (
       })
     }
   } catch (e) {
-    if (
-      e &&
-      e.code &&
-      (e.code === 'EndpointDisabled' || e.code === 'InvalidParameter')
-    ) {
+    if (e?.Code === 'EndpointDisabled' || e?.Code === 'InvalidParameter') {
```

Comment:
this seems wrong to me is that really how the AWS sdk works? gross.

Reply:
i checked the structure of the error object and this does seem like the most reasonable approach. there weren't any numeric error codes or anything.